src/components/Global.astro

```diff
@@ -1,51 +1,52 @@
 <style global>
 :root {
-    --main: #262260;
-    --secondary: #f55a8a;
-    --tertiary: #7de5e5;
-    --dark: #1a1229;
-    --white: #ffffff;
-    --mainfont: 'main', sans-serif;
+	--main: #262260;
+	--secondary: #f55a8a;
+	--tertiary: #7de5e5;
+	--dark: #1a1229;
+	--white: #ffffff;
+	--black: #000000;
+	--mainfont: 'main', -apple-system, BlinkMacSystemFont, Roboto, 'Helvetica Neue', sans-serif;
```

Comment:
Oh, I also added a couple fallback fonts to try before getting to the default `sans-serif`.